app/localShortcuts.js

```diff
@@ -35,17 +34,24 @@ module.exports.register = (win) => {
       ['F5', messages.SHORTCUT_ACTIVE_FRAME_RELOAD],
       ['Ctrl+F5', messages.SHORTCUT_ACTIVE_FRAME_CLEAN_RELOAD],
       ['Ctrl+F4', messages.SHORTCUT_CLOSE_FRAME],
+      ['Ctrl+U', messages.SHORTCUT_ACTIVE_FRAME_VIEW_SOURCE],
       ['Alt+D', messages.SHORTCUT_FOCUS_URL],
       ['Alt+Left', messages.SHORTCUT_ACTIVE_FRAME_BACK],
       ['Alt+Right', messages.SHORTCUT_ACTIVE_FRAME_FORWARD])
   } else if (process.env.NODE_ENV !== 'development') {
     // We're in Darwin and release or test mode...
     // We disable for development mode because Browser level dev tools copy doesn't work.
     // Workaround for #1060
-    simpleWebContentEvents.push([
-      'Cmd+C',
-      messages.SHORTCUT_ACTIVE_FRAME_COPY
-    ])
+    simpleWebContentEvents.push(
+      ['Cmd+C', messages.SHORTCUT_ACTIVE_FRAME_COPY]
+    )
+  } else {
```

Comment:
Oh boy do I feel stupid. Yes, absolutely right, should've been more careful. Rev 770596a adds Cmd+Alt+U for darwin always, and adds the Cmd+C only if not in dev-mode